cfme/tests/storage/test_volume.py

```diff
@@ -63,20 +109,16 @@ def test_storage_volume_create_cancelled_validation(appliance, provider):
         casecomponent: Cloud
     """
     volume_collection = appliance.collections.volumes
-    manager_name = '{} Cinder Manager'.format(provider.name)
-    volume_collection.create(name=fauxfactory.gen_alpha(),
-                             storage_manager=manager_name,
-                             tenant=provider.data['provisioning']['cloud_tenant'],
-                             size=STORAGE_SIZE,
-                             provider=provider,
-                             cancel=True)
+    create_volume(appliance, provider, from_manager, cancel=True)
 
     view = volume_collection.create_view(VolumeAllView)
     view.flash.assert_message('Add of new Cloud Volume was cancelled by the user')
 
 
+@pytest.mark.uncollectif(lambda appliance, from_manager: appliance.version < '5.10' and
```

Comment:
You can remove this appliance check. As we have separate branch for `5.9`.

Reply:
And how can I add those changes to 5.9, because I want to run it in 5.9 also?

Reply:
@mmojzis we are not running `master` against `5.9` but you can put changes against `5.9-breakpoint-patched` branch.

@mshriver ^^